kernel/kernel-api/src/test/java/io/delta/kernel/internal/types/JsonHandlerTestImpl.java

```diff
@@ -407,5 +409,151 @@ public ArrayValue getArray(int rowId) {
         public MapValue getMap(int rowId) {
             return (MapValue) values.get(rowId);
         }
+
+        @Override
+        public ColumnVector getChild(int ordinal) {
```

Comment:
I am inclined to remove this given it is used for tests. It is unnecessary 200+ lines of code that is not used by any tests right now.